backend/service-lib/wiki/getting_started.md

```diff
@@ -289,6 +287,52 @@ projectId=`curl -XPOST -H 'Content-Type: application/json' 'http://localhost:808
     -H "Authorization: Bearer ${ADMINTOK}" \
     -d '{ "title": "UCloud" , "principalInvestigator": "'"${USERNAME}"'" }' | jq .id -r`
 
+providerId=`curl -XPOST -H 'Content-Type: application/json' 'http://localhost:8080/api/providers' \
+    -H "Authorization: Bearer ${ADMINTOK}" \
+    -H "Project: ${projectId}" \
+    -d '{ "items": [{ "id": "ucloud", "domain": "localhost", "https": false, "port": 8080 }],
+    "type": "bulk"}' | jq ".responses[0].id" -r`
+
+provider=`curl -H 'Content-Type: application/json' "http://localhost:8080/api/providers/retrieve?id=${providerId}" \
+    -H "Authorization: Bearer ${ADMINTOK}" \
+    -H "Project: ${projectId}"`
+
+refreshToken=`echo "${provider}" | jq .refreshToken -r`
+publicKey=`echo "${provider}" | jq .publicKey -r`
+
+providerConfig="---
+app:
+  kubernetes:
+    providerRefreshToken: ${refreshToken}
+    ucloudCertificate: ${publicKey}"
+
+echo "$providerConfig" > $HOME/ucloud/$providerId-compute-config.yml
```

Comment:
Folder might not exist yet

Reply:
I intentionally left it out, since the getting started guide instructs to create the directory (and a config file) before running the script, but I will add it. I guess its better the script doesn't fail if the user skipped that step for some reason.

Reply:
`mkdir -p` should work